### ISSUE TITLE
Where the classifier (CE loss) of ensemble logits?

### ISSUE THREAD

**Issue by Guicai996** [Jangho-Kim/FFL-pytorch#1]
Hi, I found the pic.1 of paper which show us a ensemble classifier, but I can't see any CE loss about ensemble in this code. Is there a problem with my understanding?
